src/portal/src/app/project/webhook/webhook.service.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2017 VMware, Inc. All Rights Reserved.
```

Comment:
This license is not correct license.

Reply:
I'm so confused about it.I just copy the liense from `robot-account.service.ts`.
`// Copyright Project Harbor Authors`
I got this idea in `tag-retention.service.ts`. Is that right?

Reply:
Please update your code, and you will get the right one!